src/Tickets/Commerce/Gateways/Square/REST/Webhook_Endpoint.php

```diff
@@ -241,6 +274,8 @@ protected function process_webhook_event( array $event_data ) {
 		 * @param string $event_type The event type.
 		 */
 		do_action( 'tec_tickets_commerce_square_webhook_event', $event_data, $event_type );
+
+		remove_filter( 'tec_tickets_commerce_is_sandbox_mode', $callback );
```

Comment:
Same goes for this?

Reply:
Here we remove what we are doing previously.